src/backend/transactions/transactions.ts

```diff
@@ -3,20 +3,24 @@ import { EnrichedTransaction } from '../commonTypes';
 import { Transaction } from '../import/bankScraper';
 import { compareObjectsByDate } from './dates';
 
-const transactionArrayToObject = (transactions: EnrichedTransaction[]) => transactions.reduce((acc, enrichedTransaction) => {
-  acc[enrichedTransaction.hash] = enrichedTransaction;
+const unifyHash = (hash: string) => hash
+  .replace(/`/g, "'")
+  .replace(/00\dZ/, '000Z');
```

Comment:
This is the change. It is part of the hash creation, but also used when comparing hashes to support adding rules.

Reply:
Manipulating dates with string manipulation is not a very stable way to do it.

Why not use moment, something like `date.startOf('day').format()` with the format of your choice?

Reply:
That is because I'm starting with just hash. If I was needed to generate the hash in a better way, for sure I did your suggestion. But here I want also to unify already created hashes from existing Google Sheet, for example, and I can't regenrate the hash based on the sheet data.